tests/__snapshots__/index.test.js.snap

```diff
@@ -1763,6 +1763,137 @@ exports[`createbom produces a BOM in JSON format that includes hashes from packa
 }"
 `;
 
+exports[`createbom produces a BOM when all dependencies are dev-dependencies that shall be listed 1`] = `
+"{
+  \\"bomFormat\\": \\"CycloneDX\\",
+  \\"specVersion\\": \\"1.3\\",
+  \\"version\\": 1,
+  \\"metadata\\": {
+    \\"timestamp\\": \\"2020-01-01T01:00:00.000Z\\",
+    \\"tools\\": [
+      {
+        \\"vendor\\": \\"CycloneDX\\",
+        \\"name\\": \\"Node.js module\\",
+        \\"version\\": \\"3.0.0\\"
+      }
+    ],
+    \\"component\\": {
+      \\"type\\": \\"library\\",
+      \\"bom-ref\\": \\"pkg:npm/with-dev-dependencies@0.0.0\\",
+      \\"name\\": \\"with-dev-dependencies\\",
+      \\"version\\": \\"0.0.0\\",
+      \\"description\\": \\"test env to see if dev-dependencies are detected as such\\",
+      \\"purl\\": \\"pkg:npm/with-dev-dependencies@0.0.0\\"
+    }
+  },
+  \\"components\\": [
```

Comment:
two components.
all were dev-dependencies, dev deps were wanted to be listed.
one was a transitive dev-dep, the other a direct one

everything as expected